examples/connectivity/plot_extract_regions_dictlearn_maps.py

```diff
@@ -1,13 +1,13 @@
 """
-Regions extraction using Canonical ICA maps and functional connectomes
-======================================================================
+Regions extraction using Dictionary Learning and functional connectomes
+=======================================================================
 
 This example shows how to use :class:`nilearn.regions.RegionExtractor`
-to extract connected brain regions from whole brain ICA maps and
-use them to estimate a connectome.
+to extract connected brain regions from whole brain maps decomposed
+using dictionary learning and use them to build a functional connectome.
```

Comment:
You should add a comment here that the example would be almost the same if the maps had been computed with an ICA.

Such a comment is Google bait: people googling for "regions extraction ICA" with thus find your page